src/test/components/ZipFileTree.test.js

```diff
@@ -75,37 +74,36 @@ describe('calltree/ZipFileTree', function() {
         );
 
       return {
-        component,
-        store,
-        getState,
-        dispatch,
+        ...setupResult,
         profile1OpenLink,
         waitUntilDoneProcessingZip,
       };
     };
 
     it('starts out without any path in the zip file', async () => {
-      const { getState, component } = await setupClickingTest();
+      const { getState, container } = await setupClickingTest();
       expect(UrlStateSelectors.getPathInZipFileFromUrl(getState())).toBe(null);
-      expect(component.find(ProfileViewer).length).toBe(0);
+      expect(container.querySelector('.profileViewer')).toBeFalsy();
```

Comment:
Optional: Running a selector multiple times in the tests like this could be a bit brittle. It exposes the tests to the implementation details. I think it would be a bit more future-proof to collect the selector in the `setup` function, and expose it via a function.